packages/plugin-baby/README.md

```diff
@@ -0,0 +1,52 @@
+# Plugin helpers
+
+Set of helpers based on the inversify.js that enable implementing a plugin architecture in your app.
+
+## EventedContainer
+
+Tiny wrapper on top of inversify.js Container that allow you to subscribe to the bind, unbind and rebind events.
```

Comment:
I feel like I miss the reason why I would like to subscribe to those events. Maybe that would be interesting to mention on this doc.